airflow/models/base.py

```diff
@@ -25,9 +25,23 @@
 
 SQL_ALCHEMY_SCHEMA = conf.get("database", "SQL_ALCHEMY_SCHEMA")
 
+# For more information about what the tokens in the naming convention
+# below mean, see:
+# https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.MetaData.params.naming_convention
+naming_convention = {
+    "ix": "idx_%(column_0_N_label)s",
+    "uq": "%(table_name)s_%(column_0_N_name)s_uq",
+    "ck": "ck_%(table_name)s_%(constraint_name)s",
+    "fk": "%(table_name)s_%(column_0_name)s_%(referred_table_name)s_fkey",
+    "pk": "%(table_name)s_pkey",
+}
+
 metadata = (
-    None if not SQL_ALCHEMY_SCHEMA or SQL_ALCHEMY_SCHEMA.isspace() else MetaData(schema=SQL_ALCHEMY_SCHEMA)
+    MetaData(naming_convention=naming_convention)
+    if not SQL_ALCHEMY_SCHEMA or SQL_ALCHEMY_SCHEMA.isspace()
+    else MetaData(schema=SQL_ALCHEMY_SCHEMA, naming_convention=naming_convention)
 )
+
```

Comment:
I think this can be simplified:

```suggestion
if not SQL_ALCHEMY_SCHEMA or SQL_ALCHEMY_SCHEMA.isspace():
    SQL_ALCHEMY_SCHEMA = None
metadata =  MetaData(schema=SQL_ALCHEMY_SCHEMA, naming_convention=naming_convention)
```